### PR TITLE
[WIP] Adds the makeshift centrifuge and possibly other things

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
This pr's goal is to add the content of chemistry to maintenance and make it possibly more accessible.

## Why It's Good For The Game
A way to separate reagents without the use of a chem master as to have gradual "tiers". Although the current state of the centrifuge is not especially easy to use this is partly intended. The same way the handheld grinder is easy to make but not easy to use. 

## Changelog
:cl:
add:Adds a craftable makeshift centrifuge
/:cl:

### CODE REVIEW

**Comment by maxymax13** [comment]
soon medbay will be able to operate without a chemist entirely, sourcing its medicine entirely from a dodgy crack stand set up in the vacant comissionary from 3 assistants and some lizard kidnapped from lavaland that doesn't speak english, with some toilet paper and a mortar & pestle

**Comment by spessbro** [comment]
> soon medbay will be able to operate without a chemist entirely, sourcing its medicine entirely from a dodgy crack stand set up in the vacant comissionary from 3 assistants and some lizard kidnapped from lavaland that doesn't speak english, with some toilet paper and a mortar & pestle

you joke but that's exactly what i want to do

File: code/modules/reagents/chemistry/machinery/centrifuge.dm [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	beaker1_overlay.pixel_x = 2
+	beaker2_overlay = mutable_appearance(icon, "disp_beaker")
+	beaker2_overlay.pixel_y = -6
+	beaker2_overlay.pixel_x = 6
+	. = ..()
+
+/obj/machinery/centrifuge/proc/separate(uptime)
+	if(!length(reagents.reagent_list))
+		return FALSE
+	var/units_transfered = uptime/1 SECONDS*CENTRIFUGE_UPS
```
Comment (Qustinnus) [inline code/modules/reagents/chemistry/machinery/centrifuge.dm:39]:
```suggestion
	var/units_transfered = uptime * CENTRIFUGE_UNITS_SEPARATED_PER_SECOND * 0.1 //Converted from DS to seconds.
```

**Comment by spessbro** [comment]
will reopen and work on this when i get better

**Pull request closed without merging**